src/main/resources/application.yml

```diff
@@ -1,9 +1,9 @@
 spring:
   datasource:
     driver-class-name: com.mysql.cj.jdbc.Driver
-    password: ${MYSQL_PASSWORD}
-    url: ${MYSQL_URL}
-    username: ${MYSQL_USERNAME}
+    password: olgaolga
```

Comment:
Use enviroment variable for username, password and url:
password: ${MYSQL_PASSWORD}
url: ${MYSQL_URL}
username: ${MYSQL_USERNAME}